lib/aytests/image_builder.rb

```diff
@@ -65,7 +66,7 @@ def install(autoinst, iso_url)
       setup_iso(iso_url)
       setup_autoinst(autoinst)
       setup_definition(:install)
-      build
+      build(build_environment(autoinst))
```

Comment:
Because the same thing happens again, `build` is always called with `build_environment` as only parameter (and even the name of the method seems to indicate that it really only makes sense in the context of a build).